<launch>
    <!-- <node name="rvizvisualisation" pkg="rviz" type="rviz" output="log"
          args="-d $(find wtr_race)/config/decaymap.rviz"/> -->

    <node pkg="tf" type="static_transform_publisher" name="camera_init_to_world" args="0 0 0 0 0 0 world camera_init 40" />

    <!-- <node pkg="wtr_race" type="decaymap_detect" name="decay_map_test" output="screen"> -->
    <node pkg="wtr_race" type="decaymap_detect" name="decay_map_test" >
        <rosparam command="load" file="$(find wtr_race)/config/decaymap.yaml"></rosparam>
    </node>

</launch>
